web_src/css/base.css

```
@@ -847,6 +848,7 @@ a.label,
 
 .ui.table > thead > tr > th {
   background: var(--color-box-header);
+  border-color: var(--color-secondary-alpha-50);
```

```suggestion
  border-color: var(--color-secondary);
```

Wo don't really use alpha-channel borders anywhere else really so I think this would fit better.

---

It is the one used for table body borders through (line 822). I thought it should stay as it is there, but I have no problem with changing it

---

Right, but in any case, a thicker border for `th` is always appreciated I guess.